src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -1854,7 +1919,9 @@ def _render_frontmatter_details(frontmatter_html: str) -> str:
 
 
 def _render_lineage_card(
-    lineage: dict | None, *, requested_pack: str = "",
+    lineage: dict | None,
+    *,
+    requested_pack: str = "",
```

Ruff (0.15.12)

[warning] Unused function argument: `requested_pack`

(ARG001)